docker/build-bas-sitemap.sh

```diff
@@ -23,9 +22,8 @@ do # A paged GraphQL query, against the datacite API. ror.org/01rhff309 is the O
         -H 'Connection: keep-alive' \
         -H 'DNT: 1' \
         -H 'Origin: https://api.datacite.org' \
-        --compressed \
         --data-binary   "{\"query\":\"{\n  query: \n    organization(id: \\\"ror.org/01rhff309\\\") {\n      datasets(after: ${cursor}) {\n        totalCount\n        nodes {\n          id\n        }\n      pageInfo {\n        endCursor\n        hasNextPage\n      }\n      }\n    }\n  }\n\"}"
-
+        --compressed \
```

Comment:
This either needs to go on the previous line or the `\` needs to go at the end of the line before this one. Here, the `\` tells `bash` that there is more command on the next line; it's just put on multiple lines for readability.

Reply:
That's true. Thanks !!